docs/cli_README.md

```diff
@@ -5,28 +5,64 @@
 ### Installation:
 - Select, Download and Install latest binary from [Releases](https://github.com/fabric8-analytics/cli-tools/releases)
 
+#### curl
+
+- ##### For Linux
+```
+$. curl -s -L https://github.com/fabric8-analytics/cli-tools/releases/download/v0.0.1/crda_0.0.1_Linux_64bit.tar.gz | tar xvz -C .
+```
+- ##### For Linux - Fedora/CentOS/RHEL
+```
+$. curl -s -L https://github.com/fabric8-analytics/cli-tools/releases/download/v0.0.1/crda_0.0.1_Linux-64bit.rpm | tar xvz -C .
+```
+- ##### For MacOS
+```
+$. curl -s -L https://github.com/fabric8-analytics/cli-tools/releases/download/v0.0.1/crda_0.0.1_macOS_64bit.tar.gz | tar xvz -C .
+```
+- ##### For MacOS - Apple Silicon
+```
+$. curl -s -L https://github.com/fabric8-analytics/cli-tools/releases/download/v0.0.1/crda_0.0.1_macOS_ARM64.tar.gz | tar xvz -C .
+```
+- ##### For Windows
+```
+$. curl -s -L https://github.com/fabric8-analytics/cli-tools/releases/download/v0.0.1/crda_0.0.1_Windows_64bit.tar.gz | tar xvz -C .
```

Comment:
Windows is not as simple as this, it won't have curl or tar by default unless your are on WSL.

Reply:
Updated with direct download link